index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Rabia and Festus product page</title>
+    <link rel="stylesheet" href="style.css">
+
+  </head>
+
+<body>
+  
+  <header id="header">
```

Comment:
The header tag is a semantic tag in HTML and it's not appropriate to be used as an ID name.